draft-ietf-mls-architecture.md

```diff
@@ -1829,8 +1829,8 @@ type. In contrast, when MLS clients use the `basic` Credential type, a larger
 degree of trust must be placed in a (likely) centralized authentication
 resource, or on out-of-band processes such as manual verification.
 
-> **RECOMMENDATION:** Select the strongest MLS Credential type available among
-> the target members of an MLS group.
+> **RECOMMENDATION:** Select the strongest MLS Credential type which is
```

Comment:
```suggestion
> **RECOMMENDATION:** Select the MLS Credential type with the strongest security parameter which is
```

Reply:
What do you mean by "security parameter" in this case? Like what's the security parameter of basic?

Reply:
"security parameter" == "level of security" -> the number of security bits of the signature/authentication scheme